tests/reg_tests/test_jacvecprodBWDFast.py

```diff
@@ -0,0 +1,100 @@
+# built-ins
```

Comment:
Can you change the test file name so that the part after the test is camel case (or whatever format you prefer)? so `test_jacVecProdBWDFast.py` maybe?